policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -370,6 +363,23 @@ fn is_parent_service(parent: &ParentReference) -> bool {
         .unwrap_or(false)
 }
 
+#[inline]
+pub fn route_accepted_by_service(route: &api::HttpRoute, service: &str) -> bool {
```

Comment:
nit: does this need to be `pub`? AFAICT it isn't currently used anywhere else...
```suggestion
fn route_accepted_by_service(route: &api::HttpRoute, service: &str) -> bool {
```